src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLSPTextDocumentCreatedListener.cs

```diff
@@ -50,7 +50,9 @@ internal class RazorLSPTextDocumentCreatedListener
 
             if (_lspDocumentManager is null)
             {
+#pragma warning disable CA2208 // Instantiate argument exceptions correctly
                 throw new ArgumentException("The LSP document manager should be of type " + typeof(TrackingLSPDocumentManager).FullName, nameof(_lspDocumentManager));
```

Comment:
Should do this for the other one as well. Didn't know this also needs the pragma

Reply:
Ya it's an interesting one:

![image](https://user-images.githubusercontent.com/2008729/77352043-cc653f80-6cfb-11ea-80da-8da55ca308b0.png)